<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>新建窗口</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical{ width:20
px; background:white;
border:none}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>539</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>标签2</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView::item{height:50px}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>标签3</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>361</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView::item{height:30px}</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>91</width>
     <height>361</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView{background-color:#dddddd;border:none}

QListView::item{height:50px}

QListView::item:selected{background-color:white;  color:black}</string>
   </property>
   <item>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>会话窗口</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
